klippy/configfile.py

```diff
@@ -210,6 +252,15 @@ def set(self, section, option, value):
         logging.info("save_config: set [%s] %s = %s", section, option, svalue)
     def remove_section(self, section):
         self.autosave.fileconfig.remove_section(section)
+    def _disallow_include_conflicts(self, regular_data, cfgname, gcode):
+        config = self._build_config_wrapper(regular_data, cfgname)
+        for section in self.autosave.fileconfig.sections():
+            for option in self.autosave.fileconfig.options(section):
+                if config.fileconfig.has_option(section, option):
+                    msg = "SAVE_CONFIG section '%s' option '%s' conflicts " \
+                          " with included value" % (section, option)
+                    logging.exception(msg)
```

Comment:
I think logging.exception() should only be called from an exception handler.   The following `raise gcode.error()` will already log the error message, so it shouldn't be necessary to explicitly log it from here.

Reply:
Fixed.  Good that it works this way, I was actually emulating the way exceptions were reported in cmd_SAVE_CONFIG.  You want me to remove the other extra logging while I'm in there?